app/services/preserver/blind_importer.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+class Preserver::BlindImporter
+  def self.import(id:, change_set_persister:, source_metadata_adapter: default_source_metadata_adapter)
+    new(id: id, change_set_persister: change_set_persister, source_metadata_adapter: source_metadata_adapter).import!
+  end
+
+  def self.default_source_metadata_adapter
+    FileMetadataAdapter.new(storage_adapter: source_storage_adapter)
+  end
+
+  def self.source_storage_adapter
+    Valkyrie::StorageAdapter.find(:google_cloud_storage)
+  end
+
+  attr_reader :id, :source_metadata_adapter, :change_set_persister
+  delegate :storage_adapter, to: :change_set_persister
+  def initialize(id:, source_metadata_adapter:, change_set_persister:)
+    @id = id
+    @source_metadata_adapter = source_metadata_adapter
+    @change_set_persister = change_set_persister
+  end
+
+  # Imports the given ID from the source metadata adapter as well as all its
+  # children.
+  def import!
+    import_binary_files
+    member_ids = source_resource.try(:member_ids) || []
+    member_ids.map! do |member_id|
```

Comment:
So after mapping from the figgy ids to the cloud storage ids in the query service, now we're mapping back to figgy ids. 👍